contracts/SnappAuction.sol

```diff
@@ -157,22 +157,10 @@ contract SnappAuction is SnappBase {
         uint8 sellToken,
         uint96 buyAmount,
         uint96 sellAmount
-    ) public onlyRegistered() {
-        createNewPendingBatchIfNecessary();
-
-        // Update Auction Hash based on request
-        uint16 accountId = publicKeyToAccountMap(msg.sender);
-        bytes32 nextAuctionHash = sha256(
-            abi.encodePacked(
-                auctions[auctionIndex].shaHash,
-                encodeOrder(accountId, buyToken, sellToken, buyAmount, sellAmount)
-            )
-        );
-        auctions[auctionIndex].shaHash = nextAuctionHash;
-
-        emit SellOrder(auctionIndex, auctions[auctionIndex].size, accountId, buyToken, sellToken, buyAmount, sellAmount);
-        // Only increment size after event (so it is emitted as an index)
-        auctions[auctionIndex].size++;
+    ) public {
+        // Ignore first 4 bytes padding and last two bytes accountId
```

Comment:
Why are the first four bytes ignored here? I don't get it

Reply:
Also, I suppose this accountId will become 3 ( = 24 /8) bytes. Does this imply some change needs to be made in #124 ?

Reply:
Check `encodeOrder`. We create a bytes32 out of 28 bytes. This means there is 4 bytes of padding.

It will become 3 bytes with #124